indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -316,12 +366,13 @@ <h3 class="abstract-title">
                 <div class="i-timeline-item-label action"></div>
                 <div class="i-timeline-item-box header-indicator">
                     <div class="i-box-header">
-                        {% trans name=abstract.submitter.name, date=abstract.submitted_dt|format_human_date -%}
-                            <strong>{{ name }}</strong> submitted this abstract on {{ date }}
+                        {% trans name=abstract.submitter.name -%}
+                            <strong>{{ name }}</strong> submitted this abstract
                         {%- endtrans %}
                         {% if abstract.modified_dt %}
                             {{ render_edited_hint(abstract.modified_dt, abstract.modified_by) }}
                         {% endif %}
+                        <time>{{ abstract.submitted_dt|format_human_date }}</time>
```

Comment:
Does this still look good if there's also a `modified_dt`? Also, `render_edited_hint` could probably make use of `<time>` too